src/saving_tools.jl

```diff
@@ -112,10 +112,7 @@ function read_stdout_stderr(cmd::Cmd)
     return (exception = exception, out=read(out,String), err=read(err,String))
 end
 
-"""
-    gitpatch(gitpath = projectdir())
-
-Generates a patch describing the changes of a dirty repository
+""" Generates a patch describing the changes of a dirty repository
```

Comment:
I am confused about this documentation string change. It is a mistake. The previous version was obviously clearer.